### ISSUE TITLE
[a11y] Users are able to set passwords that do not meet the requirements while using the keyboard

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5053]
**Important details:**

* This is not reproducible if the VoiceOver/Screen Reader is not enabled, as the “Create account” button cannot be selected if the quick nav is not active;

* Not reproducible if using the “Space” on “Enter” key while the “Create account” button is in focus;

**Affected versions:**

* Mozilla VPN 2.12.0 (2.202211171941); 

* Mozilla VPN 2.12.0 (2.202211171926);

**Tested Platforms:**

* MacOS 11.6;

* Windows 11;

**Prerequisites:**

* Have the Voice Over enabled (MacOS: Apple menu > System Preferences > Accessibility > VoiceOver > Check "Enable VoiceOver") or other Screen reader opened;

* Have Mozilla VPN app installed and opened;

* Be in Email screen;

**Steps to reproduce:**

1. Enter an email address that is not associated with a Fx account and press the “Continue” button;
1. In the password field enter 2 random chars / enter email address / common password;
1. Press the right arrow key until the “Create account” button gains focus;
1. Press Control+Option+Space keys on MacOS / Caps+Enter keys on Windows;
1. Observe the behavior;

**Expected result:**

* The user remains in the Password creation screen, as the requirements are not met and the “Create account” button is disabled;

**Actual result:**

* The user is redirected to the Verification code screen.

**Notes:**

* A verification code is received on email, and the user is able to proceed with the verification process;

* After verifying the account, the user is redirected to the “Subscribe to Mozilla VPN” screen;

* See the attached video and logs;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-3388)
┆Reporter: Bianca Hidecuti

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Still reproducing with the 2.17 version.

**Comment by data-sync-user** [comment]
➤ Matt Cleinman commented:

Cannot replicate - This was fixed by  in a recent PR: https://github.com/mozilla-mobile/mozilla-vpn-client/commit/c080b619543fb310ced173b0d7fdaf4cb8dd1031#diff-2dc0cb2dbe3407a9161e1b0c268ea587087af40b1fa3f18903c25ee7386b5f33R47

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Verified this on Mozilla VPN 2.19.0 (2.202311132237), on macOS 12.6.6 and on Windows 11 and the issue is no longer reproducible. Marking this verified as fixed. 

Matt Cleinman, why was this set to “Cancelled - Cannot reproduce” since the PR mentioned by you fixed the issue? Thanks!
